tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -1148,3 +1148,8 @@ test_i2c_npm2100: npm2100@9f {
 		compatible = "nordic,npm2100-vbat";
 	};
 };
+
+test_i2c_hs400x: hs400x@A0 {
+	compatible = "renesas,hs400x";
+	reg = <0xA0;
+};
```

Comment:
```suggestion
test_i2c_hs400x: hs400x@a0 {
	compatible = "renesas,hs400x";
	reg = <0xa0>;
};
```

Reply:
Adjusted, thanks! I totally overlooked the capitalization and missing > yesterday